README.md

```diff
@@ -24,9 +24,9 @@ SDK for building multiplayer HTML5 games that run inside Rune on [Android](https
+| -------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------- |:---------------------------------------------------------------------------------------------------------------------------------------------------|
 | [<img src="docs/static/img/multiplayer-games/tic-tac-toe.png" width=500>](https://developers.rune.ai/examples/tic-tac-toe/)                              | [<img src="docs/static/img/multiplayer-games/outmatched.png" width=500>](https://developers.rune.ai/examples/outmatched/)                              | [<img src="docs/static/img/multiplayer-games/sudoku.png" width=500>](https://developers.rune.ai/examples/sudoku/)                                   | [<img src="docs/static/img/multiplayer-games/pinpoint.png" width=500>](https://developers.rune.ai/examples/pinpoint/)                              |
-| [Demo](https://developers.rune.ai/examples/tic-tac-toe/), [Source](https://github.com/rune/rune-games-sdk/tree/staging/multiplayer/examples/tic-tac-toe) | [Demo](https://developers.rune.ai/examples/outmatched/), [Source](https://github.com/rune/rune-games-sdk/tree/staging/multiplayer/examples/outmatched) | [Demo](https://developers.rune.ai/examples/tic-tac-toe/), [Source](https://github.com/rune/rune-games-sdk/tree/staging/multiplayer/examples/sudoku) | [Demo](https://developers.rune.ai/examples/pinpoint/), [Source](https://github.com/rune/rune-games-sdk/tree/staging/multiplayer/examples/pinpoint) |
+| [Demo](https://developers.rune.ai/multiplayer/examples/tic-tac-toe/), [Source](https://github.com/rune/rune-games-sdk/tree/staging/examples/tic-tac-toe) | [Demo](https://developers.rune.ai/multiplayer/examples/outmatched/), [Source](https://github.com/rune/rune-games-sdk/tree/staging/examples/outmatched) | [Demo](https://developers.rune.ai/multiplayer/examples/tic-tac-toe/), [Source](https://github.com/rune/rune-games-sdk/tree/staging/examples/sudoku) | [Demo](https://developers.rune.ai/multiplayer/examples/pinpoint/), [Source](https://github.com/rune/rune-games-sdk/tree/staging/examples/pinpoint) |
```

Comment:
Is the path here correct to the demos?

Reply:
Yes, until we change the docs to no longer have singleplayer docs. We could consider also doing that now I guess 🤠

Reply:
Ah, nvm. You're right 👍

Reply:
Fixed in https://github.com/rune/rune-games-sdk/pull/167/commits/277961757cb0cd1f3186c583eaca938cb817360b.